sdks/java/io/fileschematransform/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://s.apache.org/beam-owners
+
+reviewers:
+  - johnjcasey
+  - pabloem
```

Comment:
@johnjcasey and @pabloem Are you both ok being listed under OWNERS and should others be included?